game-core/src/main/java/games/strategy/engine/framework/ServerGame.java

```diff
@@ -158,6 +162,27 @@ public void shutDown() {}
     remoteMessenger.registerRemote(serverRemote, SERVER_REMOTE);
   }
 
+  private void importDiceStats(HistoryNode node)  {
+    if (node instanceof EventChild) {
+      EventChild childNode = (EventChild) node;
+      if (childNode.getRenderingData() instanceof DiceRoll) {
+        String playerName = DiceRoll.getPlayerNameFromAnnotation(childNode.getTitle());
+        PlayerID playerId = gameData.getPlayerList().getPlayerId(playerName);
+
+        DiceRoll diceRoll = (DiceRoll) childNode.getRenderingData();
+        int[] rolls = new int[diceRoll.size()];
```

Comment:
As mentioned earlier, we have this weird style guideline to make everything final that's not being changed.

Reply:
Done.